src/opnsense/mvc/app/library/OPNsense/Backup/GDrive.php

```diff
@@ -192,7 +192,7 @@ public function backup()
                 try {
                     $client = new \Google\API\Drive();
                     $client->login(
-                        (string)$config->system->remotebackup->GDriveEmail,
+                        (string)$config->system->remotebackup->GDriveUID,
```

Comment:
this will break existing setups, as the address/id is currently stored here.

Reply:
But only changing the label would be fine? Unfortunately I am not a coder.

Reply:
if their documentation refers to uid now instead of email, I'm fine with changing the label, I haven't checked myself yet to be honest.

Reply:
I also did a pull request for the docs here: https://github.com/opnsense/docs/pull/586